src/tribler-core/tribler_core/modules/ipv8_module_catalog.py

```diff
@@ -109,8 +107,8 @@ class PopularityCommunityLauncher(IPv8CommunityLauncher):
 @set_in_session('gigachannel_community')
 @overlay('tribler_core.modules.metadata_store.community.gigachannel_community', 'GigaChannelCommunity')
 @kwargs(metadata_store='session.mds', notifier='session.notifier')
-@walk_strategy('ipv8.peerdiscovery.discovery', 'RandomWalk')
-@walk_strategy('tribler_core.modules.metadata_store.community.sync_strategy', 'SyncChannels', target_peers=INFINITE)
+@walk_strategy('ipv8.peerdiscovery.discovery', 'RandomWalk', target_peers=30)
```

Comment:
Well, the only thing I can come up with is:
```python

# GigaChannel Community search works better with a higher than average amount of peers
GIGACHANNEL_COMMUNITY_TARGET_PEERS = 30

...
 @walk_strategy('ipv8.peerdiscovery.discovery', 'RandomWalk', target_peers=GIGACHANNEL_COMMUNITY_TARGET_PEERS) 
...

```

Does it improve readability for you?

Reply:
Now, imagine the same file (`ipv8_module_catalog.py`) where *all* the values are done in this style. That will effectively double the amount of code in the file, creating lines of repeated comments. Would *that* increase readability?

Reply:
> Does it improve readability for you?

In this case, it is not about readability, but about understandability.
Clearer now why you use `30`

> Would that increase readability

It will save you from typing errors like `'ipv8.peerdiscvery.discovery` and save your time in case of renaming.